Signal/src/ViewControllers/HomeView/ConversationSearchViewController.swift

```diff
@@ -429,5 +428,8 @@ class EmptySearchResultCell: UITableViewCell {
         let format = NSLocalizedString("HOME_VIEW_SEARCH_NO_RESULTS_FORMAT", comment: "Format string when search returns no results. Embeds {{search term}}")
         let messageText: String = NSString(format: format as NSString, searchText) as String
         self.messageLabel.text = messageText
+
+        messageLabel.textColor = Theme.primaryColor
+        messageLabel.font = UIFont.ows_dynamicTypeBody
```

Comment:
This is totally unrelated, right?

Reply:
Yup - unrelated, just noticed while testing that the cell text was not visible in dark theme.